dod-advana/gamechanger-web
# Task/jbook portfolio builder

## Description

Created a Portfolio Builder frontend + 1 basic fetch API.
the portfolio builder can be found in the sidenav of the JBOOK admin page

## !vibez
( ˘▽˘)っ♨

## Related Issue/Ticket



https://jira.di2e.net/browse/UOT-138231

## Types of changes

## Smoke testing instructions



- navigate to /#/jbook -> admin panel -> portfolio builder (the wrench icon and then the folder icon in sidenav)
- check to see the jaic portfolio exists, I've also added 4 dummy portfolios to show what it would look like with more.
- check for the look and feel to match mocks:  https://xd.adobe.com/view/d9f0fc48-972d-4bb2-b0f5-5ac6b49646a2-8ca3/screen/90ca55ed-eea7-4e9d-9c4f-b112db69ca81

## Checklist:
